rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/ScheduleAdminController.java

```diff
@@ -59,4 +59,76 @@ public ResponseEntity<?> unscheduleAllServices()throws TException {
         HttpStatus status = HttpStatus.OK;
         return new ResponseEntity<>(requestStatus, status);
     }
+
+    @RequestMapping(value = SCHEDULE_URL + "/scheduleSvmSync", method = RequestMethod.POST)
+    public ResponseEntity<?> scheduleSvmSync()throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        RequestSummary requestSummary = scheduleService.svmSync(sw360User);
+        HttpStatus status = HttpStatus.OK;
```

Comment:
Please consider using `202 ACCEPTED` instead of `200 OK` as no immediate action is taken for all endpoints.